src/plugins/partner_sku_reviews.coffee

```diff
@@ -1144,12 +1159,33 @@ class PartnerSkuReviews
       margin: 2px 0;
     }
 
+    ##{flavor}-product-reviews-extended .sa-verification-mark {
+      font-size: 11px;
+      line-height: 20px;
+      color: #707070;
+    }
+
+    ##{flavor}-product-reviews-extended .sa-verification-mark:before {
+      content: url("data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iMTRweCIgaGVpZ2h0PSIxNHB4IiBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCA4MCA4MCIgdmVyc2lvbj0iMS4xIiB2aWV3Qm94PSIwIDAgODAgODAiIHhtbDpzcGFjZT0icHJlc2VydmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxnIGZpbGw9IiM1Y2I4NWMiPgoJPHBvbHlnb24gcG9pbnRzPSIyNS42IDMzLjggMjAuMSAzOS42IDM1LjUgNTQuMSA3Ni42IDE0LjkgNzEuMSA5LjEgMzUuNSA0My4xIi8+Cgk8cGF0aCBkPSJtNjQuOCAzNWMwLjQgMS44IDAuNiAzLjYgMC42IDUuNSAwIDE0LjctMTEuOSAyNi42LTI2LjYgMjYuNnMtMjYuNy0xMS45LTI2LjctMjYuNiAxMS45LTI2LjYgMjYuNi0yNi42YzUuMyAwIDEwLjMgMS42IDE0LjUgNC4zbDUuNi01LjljLTUuNy00LTEyLjYtNi40LTIwLjEtNi40LTE5LjEgMC0zNC42IDE1LjUtMzQuNiAzNC42czE1LjUgMzQuNiAzNC42IDM0LjYgMzQuNi0xNS41IDM0LjYtMzQuNmMwLTQuMy0wLjgtOC40LTIuMi0xMi4ybC02LjMgNi43eiIvPgo8L2c+Cjwvc3ZnPgo=");
+      vertical-align: middle;
+    }
+
     ##{flavor}-product-reviews-extended .sa-review-info > *,
     ##{flavor}-product-reviews-extended .sa-authorship-info > * {
       display: inline;
       vertical-align: middle;
     }
 
+    ##{flavor}-product-reviews-extended .sa-verification-wrap {
+      padding-top: 5px;
+      display: inline-block;
+      float: right;
```

Comment:
A general note: prefer using flexbox instead of `float: right` whenever you want 2 elements attached to the sides and maximum space between them. It will save you from trouble in many cases as it does not affect and it is not affected by other elements (and it always works).